core/iwasm/interpreter/wasm_loader.c

```diff
@@ -415,6 +415,22 @@ const_str_list_insert(const uint8 *str, uint32 len, WASMModule *module,
     return node->str;
 }
 
+
```

Comment:
Should also apply same changes to wasm_mini_loader.c, which can be compiled with 'cmake -DWAMR_BUILD_MINI_LOADER=1`.